rust/lance-index/src/scalar/inverted/index.rs

```diff
@@ -761,6 +761,31 @@ impl TokenSet {
         }
     }
 
+    // the `removed_token_ids` must be sorted
+    pub fn remap(&mut self, removed_token_ids: &[u32]) {
+        if removed_token_ids.is_empty() {
+            return;
+        }
+
+        if let TokenMap::HashMap(ref mut map) = self.tokens {
```

Comment:
nit: we can use `let TokenMap::HashMap(ref mut map) = self.tokens else { return; }` here for better reading.